jasmine/files/test.json

```diff
@@ -0,0 +1,10 @@
+[{
```

Comment:
### Nitpick

Please ensure you use the following format when declaring multiple objects in a collection:
```
[
   {
      x: 'value'
   },
   {
      y: 'value'
   }
]
```